### ISSUE TITLE
Incorrect switch if same file name exists elsewhere

### ISSUE THREAD

**Issue by tophmatthews** [dschwen/switch-header-source#32]
When on `test/src/kernels/MatDiffusion.C`, switch goes to `modules/phase_field_include/kernels/MatDiffusion.h`

**Comment by dschwen** [comment]
Yeah, that's a tough one. How to you propose this should be handled?

**Comment by tophmatthews** [comment]
Only having one `MatDiffusion` in the repo? I'm surprised that is even "allowed".

The other way would be to check the path that differs by the `include` or `src` subdirectory first.

**Comment by dschwen** [comment]
They are never both used in the same app. The include/src check would severely limit the usefulness of the plugin (I use it to switch between .h,.C, and .md files using `(.*)(\.(h|C|md))$` as my _Tracked file regular expression`).

**Comment by tophmatthews** [comment]
They aren't used in the same app, but they're still all accessible in my atom Moose window. I didn't know you could switch with the md now, that's handy.

All in all, not really a big deal since this seems like a one-off. Just an FYI.
